.env.example

```diff
@@ -0,0 +1,11 @@
+#ALLOWED_HOSTS="*"  # <-- Uncomment this in dev
+
+COURTLISTENER_API_KEY=""
+
+BCB_MATCH_LIMIT=10
```

Comment:
I think we can remove this setting everywhere it appears.

Reply:
You're right. After cleaning up the commands this setting is not used.